src/main/scala/org/apache/mesos/chronos/scheduler/api/Iso8601JobResource.scala

```diff
@@ -10,6 +10,7 @@ import org.apache.mesos.chronos.scheduler.jobs.{ScheduleBasedJob, _}
 import com.codahale.metrics.annotation.Timed
 import com.google.common.base.Charsets
 import com.google.inject.Inject
+import org.apache.mesos.chronos.schedule.ISO8601Parser
```

Comment:
not for now - all the functions in here use the higher ScheduleBasedJob class, rather than the sub classes